doc/antora/modules/raddb/nav.adoc

```diff
@@ -67,6 +67,7 @@
 *** xref:mods-available/sqlippool.adoc[SQL-IP-Pool Module]
 *** xref:mods-available/sradutmp.adoc[sRadutmp Module]
 *** xref:mods-available/stats.adoc[Stats Module]
+*** xref:mods-available/totp.adoc[TOTP Module]
```

Comment:
you likely also need to do `git add doc/antora/..../totp.adoc`, otherwise the file won't exist.